pkg/controllermanager/controller/shoot/shoot_hibernation_types.go

```diff
@@ -103,6 +103,7 @@ func (h *hibernationJob) Run() {
 				return nil, fmt.Errorf("shoot %s/%s hibernation schedule changed mid-air", shoot.Namespace, shoot.Name)
 			}
 			shoot.Spec.Hibernation.Enabled = &h.enabled
+			shoot.Spec.Provider.EnableContainerD = true;
```

Comment:
Hmm, I don't think we can do this yet. Yes, eventually we want all the clusters to use containerd, however, we should announce this properly with a deprecation period and let people switch themselves when the time fits for them.

Reply:
Ok, i will remove it but do you want to keep it in the GEP?

Reply:
Hm, I'll come back to this question. Gimme some time.

Reply:
Hi Nimrod, yes, for now let's not change it on-the-fly. We should rather ask users to do it with a proper deprecation period.

Reply:
Ok, i will remove the true defaults from the code and the GEP